dashboard/modules/log/log_agent.py

```diff
@@ -296,7 +296,7 @@ async def ListLogs(self, request, context):
             )
         log_files = []
         for p in path.glob(request.glob_filter):
-            log_files.append(p.name)
+            log_files.append(str(p.relative_to(path)))
```

Comment:
This is the main change. We don't include subdirectory parent directories before, so the list results are not streamable.